src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateMethodService.State.cs

```diff
@@ -242,7 +242,7 @@ private static ImmutableArray<string> GenerateParameterNamesBasedOnParameterType
             }
 
             NameGenerator.EnsureUniquenessInPlace(names, isFixed);
-            return names.ToImmutable();
+            return names.ToImmutableAndClear();
```

Comment:
>names [](http://example.com/codeflow?start=19&length=5)

nit: another

Reply:
couldn't do this one.

Reply:
Oh, I'm assuming EnsureUniquenessInPlace method mucks it up, although I'm not clear why.

Reply:
well, we'd need to make EUIP take one of these collectoins. and update hte collection for whatever support this helper needs.  i'm trying to keep things small in scope :)